Update subscriptions.md - some visual improvements (#5979)
harness/developer-hub: docs/platform/get-started/subscriptions.md

#### Continuous Delivery & GitOps (CD)

> **Services**: CD module deploys software services, on to infrastructure platforms spanning traditional VMs, Kubernetes, public cloud platforms, Serverless functions and other custom deployment targets. A Service is an independent unit of software you track & manage through Harness CD & GitOps. This will typically map to:

Reviewer: ```suggestion
**Services:** The CD module deploys software services onto infrastructure platforms spanning traditional VMs, Kubernetes, public cloud platforms, Serverless functions, and other custom deployment targets. A Service is an independent unit of software you track & manage through Harness CD & GitOps. This will typically map to:
```

#### Feature Flags (FF)

> **Developers & Monthly Active Users (MAUs)**: FF is a feature flag rollout and management module that tracks Developers based on their active engagement with the FF module. Every platform user that actively creates / edits / deletes a flag or a group, is tracked as a FF developer. MAUs or Monthly Active Users represent the unique users seen every month, from one or more client side applications that evaluate various flags managed via Harness.

Reviewer: ```suggestion
**Developers & Monthly Active Users (MAUs):** FF is a feature flag rollout and management module that tracks Developers based on their active engagement with the FF module. Every platform user that actively creates / edits / deletes a flag or a group, is tracked as a FF developer. MAUs or Monthly Active Users represent the unique users seen every month, from one or more client side applications that evaluate various flags managed via Harness.
```

FF tracks ‘Developers’ and ‘MAU’ license consumption.

#### Service Reliability Management (SRM)

> **Services**: SRM helps manage SLOs/SLIs for various services in a R&D organization. A Service is an independent unit of software you track & manage through Harness SRM. This will typically map to 

Reviewer: ```suggestion
**Services:** SRM helps manage SLOs/SLIs for various services in an R&D organization. A Service is an independent unit of software you track & manage through Harness SRM. This will typically map to 
```

#### Chaos Engineering (CE)

> **Services**: CE helps run chaos experiments across various services in a R&D organization, towards the goal of making them resilient . A Service is an independent unit of software you track & manage through Harness CD & GitOps. This will typically map to 

Reviewer: ```suggestion
**Services:** CE helps run chaos experiments across various services in an R&D organization, with the goal of making them resilient. A Service is an independent unit of software you track & manage through Harness CD & GitOps. This will typically map to 
```

#### Infrastructure as Code Management (IACM)

> **Executions**: Every successful IACM stage execution that uses an Infrastructure Provider’s `Apply’ command (such as terraform apply) that results in resource changes, is counted as an execution.

Reviewer: ```suggestion
**Executions:** Every successful IACM stage execution that uses an Infrastructure Provider’s `Apply’ command (such as terraform apply) that results in resource changes is counted as an execution.
```

IACM tracks ‘Executions’ license consumption, instead of Developers.

#### Security Test Orchestration (STO)

> **Security Scans**: A Security Scan is defined as the execution of the STO step within a pipeline, tracked over a last 30 day active window. This involves scanning an artifact (referred to as the Target), which can be a Repository, Docker image, or a live application, for security vulnerabilities.

Reviewer: ```suggestion
**Security Scans:** A Security Scan is defined as the execution of the STO step within a pipeline, tracked over a last 30-day active window. This involves scanning an artifact (referred to as the Target), which can be a Repository, Docker image, or a live application, for security vulnerabilities.
```

STO tracks ‘Security Scans’ license consumption, instead of Developers.

#### Software Supply Chain Assurance (SSCA)

> **Supply Chain Executions**: A Supply Chain Execution is defined as the execution of SSCA step in a pipeline, tracked over a last 30 day active window. Generating SBOMs, enforcing SBOM policies, generating SLSA provenance or verifying SLSA provenance are all counted as unique SSCA steps.

Reviewer: ```suggestion
**Supply Chain Executions:** A Supply Chain Execution is defined as the execution of an SSCA step in a pipeline, tracked over a last 30-day active window. Generating SBOMs, enforcing SBOM policies, generating SLSA provenance, or verifying SLSA provenance are all counted as unique SSCA steps.
```

SSCA tracks ‘Supply Chain Executions’ license consumption, instead of Developers.

#### Cloud Cost Management (CCM)

> **Cloud Spend**: CCM is licensed on Monthly / Annual Cloud Spend that is managed and optimized via Harness. This cloud spend is actively tracked in the CCM module experience and tracked against the licensed cloud spend. As previously noted, CCM does not follow the Developer 360 subscription model.

Reviewer: ```suggestion
**Cloud Spend:** CCM is licensed on Monthly / Annual Cloud Spend that is managed and optimized via Harness. This cloud spend is actively tracked in the CCM module experience and tracked against the licensed cloud spend. As previously noted, CCM does not follow the Developer 360 subscription model.
```
